compiler/rustc_trait_selection/src/traits/query/type_op/custom.rs

```diff
@@ -82,7 +82,7 @@ where
     let value = infcx.commit_if_ok(|_| {
         let ocx = ObligationCtxt::new(infcx);
         let value = op(&ocx).map_err(|_| {
-            infcx.dcx().span_delayed_bug(span, format!("error performing operation: {name}"))
+            infcx.dcx().span_bug(span, format!("error performing operation: {name}"))
```

Comment:
theoretically reachable I think, may make sense to still keep this as is to get a test :shrug: easier to just keep a `span_delay_bug` here